packages/drivers/odsp-driver/src/fetchSnapshot.ts

```diff
@@ -696,17 +681,23 @@ export async function downloadSnapshot(
 
 	const queryString = getQueryString(queryParams);
 	const url = `${snapshotUrl}/trees/latest${queryString}`;
+	const method = "POST";
 	// The location of file can move on Spo in which case server returns 308(Permanent Redirect) error.
 	// Adding below header will make VROOM API return 404 instead of 308 and browser can intercept it.
 	// This error thrown by server will contain the new redirect location. Look at the 404 error parsing
 	// for further reference here: \packages\utils\odsp-doclib-utils\src\odspErrorUtils.ts
 	const header = { prefer: "manualredirect" };
-	const { body, headers } = getFormBodyAndHeaders(odspResolvedUrl, storageToken, header);
+	const authHeader = await getAuthHeader(
+		{ refresh: false, request: { url, method } },
```

Comment:
I'd think that we want to propagate refresh argument through the stack, as we can get into here when we need to refresh token - see my other comment in fetchSnapshot()

Reply:
Thanks for catching this, I think this is a bug. Taking a look at what could be done.

Reply:
This should be fixed now.